Iceberg-Plugin-GitHub.package/IceGitHubNewPullRequestAction.class/instance/basicExecute.st

```diff
@@ -3,22 +3,26 @@ basicExecute
 
 	credentials := IceGitHubAPI ensureCredentials.
 
-	remote ifNil: [  
+	remote ifNil: [ 
 		remote := self repository branch hasUpstream
 			ifTrue: [ self repository branch upstream remote ]
-			ifFalse: [ self askRemote: self repository ].
+			ifFalse: [ self askRemote: self repository ]
 	].
 
 	(self validateMakePullRequestOn: self repository) ifFalse: [ ^ #() ].
-		
+
 	(IceGitHubCreatePullRequestModel 
-		repository: self repository 
+		repository: self repository
 		credentials: self credentials
 		headRemote: remote)
-		setModal: true; 
+		setModal: true;
 		onAccept: [ :pullRequest | 
-			[ 
-				pullRequest send.
-				UIManager default inform: 'Pull request created.' ]
-			on: IceGitHubError do: [ :e | self reportError: e ] ];
-		openWithSpec 
\ No newline at end of file
+			[ | pullRequestDatas url |
+			pullRequestDatas := pullRequest send.
+			url := pullRequestDatas at: 'html_url'.
+			UIManager default 
+				inform: 'Pull request created. Click to view on Github.' 
+				actionOnClick: [ self class environment at: #WebBrowser ifPresent: [ :webBrowser | webBrowser openOn: url ] ifAbsent: [ self inform: ('Cannot open "{1}" because the project WebBrowser is not present by default in Pharo 6.' format: { url }) ]  ] ]
```

Comment:
It would not be necessary to check for web browser. It is included in Pharo 7 onwards

Reply:
Iceberg can be loaded in Pharo 6.1 currently.